upgrade_analysis/wizards/upgrade_generate_record_wizard.py

```diff
@@ -2,38 +2,21 @@
 # Copyright 2016 Opener B.V. <https://opener.am>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from threading import current_thread
 
 from odoo import _, fields, models
 from odoo.exceptions import UserError
 from odoo.modules.registry import Registry
 
+from ..odoo_patch.odoo_patch import OdooPatch
+
 
 class GenerateWizard(models.TransientModel):
     _name = "upgrade.generate.record.wizard"
     _description = "Upgrade Generate Record Wizard"
 
     state = fields.Selection([("draft", "Draft"), ("done", "Done")], default="draft")
 
-    # from openupgradelib import openupgrade_tools
-    # TODO, SLG, make better a patch in odoo_patch
-    # def quirk_standard_calendar_attendances(self):
```

Comment:
this part is also obsolete in V14 ?

Reply:
I could reinstall all Odoo modules with this set of patches, so yes.